### ISSUE TITLE
I can't find the sponsor button 

### ISSUE THREAD

**Issue by Luxcium** [romkatv/powerlevel10k#1235]
How can I sponsor your project (for a small amount like 5 or 10$ per month)

**Comment by Luxcium** [comment]
this issue has been closed without any fix → consider adding a github sponsor button or patreon #397

**Comment by Luxcium** [comment]
or this issue → Suggestion: GitHub Sponsor or Buy Me a Coffee #846 
###### (sorry I didn't do my research before opening this issue)

**Comment by Luxcium** [comment]
I feel I have been a little rude by opening this issue (I didn't sleep much and I thought it was obvious that this project should have a sponsor button)... I should have been more careful but regardless I would have opened the issue because your project deserves the time it would take to add a sponsor button and remember that if you receive too much money you can use it to sponsor the projects you value the most...  

Can you consider keeping this issue open if you choose not to add the sponsor button now...  and close it later if one day you have enough time to implement the thing ???

**Comment by romkatv** [comment]
> How can I sponsor your project (for a small amount like 5 or 10$ per month)

Thanks! I sincerely appreciate the gesture if I won’t be receiving the money.

Have a nice day!

**Comment by romkatv** [comment]
> I feel I have been a little rude by opening this issue

No harm done. “Please take my money” is the kind of request I don’t mind receiving 😁

> Can you consider keeping this issue open if you choose not to add the sponsor button

Sure, I can do that.  I’ll keep the issue open so that others can see it. I don’t plan on adding any options for donation.

**Comment by Luxcium** [comment]
how can I send you a private message? I sent a Twitter message ...  (I will delete this message after you reply)

**Comment by romkatv** [comment]
> how can I send you a private message? I sent a Twitter message ... (I will delete this message after you reply)

See https://github.com/romkatv/powerlevel10k#where-can-i-ask-for-help-and-report-bugs. The private option is email. Please use it only if really necessary (e.g. a security vulnerability report). If you just want to ask a question, filing an issue is the best, followed by gitter.

**Comment by nabby27** [comment]
Even if you don't want sponsors and don't like the "take my money" philosophy. Have you considered using a rewards system to encourage collaboration from external developers and also earn a small %? It's not just "take my money", but it's compensation for your effort on the specific topic and also valuing the efforts of the contributors.

I created a platform for this type of situation, [opire.dev](https://opire.dev), and the owners of the code [will receive a %](https://docs.opire.dev/rewards/pricing#percentage-for-code-owner) of rewards when created by someone outside the organization.

p10k is a solid product that I have been using for several years and I would love to be able to spend money on solving problems and also feel like I have a hand in the project.

**Comment by romkatv** [comment]
Please don't get it the wrong way but I don't actually want to encourage collaboration from external developers. Developing and maintaining a solo project is very different from managing a community. I like the former but not the latter. Keep in mind that powerlevel10k is a hobby project which is unimportant in the grand scheme of things. It's just a zsh theme.

**Comment by rwmitchell** [comment]
> powerlevel10k is a hobby project which is unimportant in the grand scheme of things.

Maybe for you, but I suspect it brings a little joy each day to everyone who uses it.  Thanks.

**Comment by romkatv** [comment]
Certainly. It also brings me joy to see others find value in my code. However, all things considered, it would be presumptuous to deem powerlevel10k as fundamentally significant.

**Comment by Syphdias** [comment]
Should we every meet, I'll get you a beverage of your choosing!
